filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -306,19 +306,23 @@ The default setting is 10s.
 [[scan-sort]]
 ===== scan.sort
 
-Specifies if files should be harvested in order and how to determine the order. Possible values are modtime, filename and none. To sort by file modification time use modtime otherwise use filename.
+[]experimental
 
-If you specify a value other than none for this setting you can determine whether to use ascending or descending order using `scan.order`.
+Specifies if files should be harvested in order and how to determine the order. Possible values are `modtime`, `filename`. Leaving the option empty means its disabled. To sort by file modification time use modtime otherwise use filename.
 
-The default setting is none.
+If you specify a value for this setting you can determine whether to use ascending or descending order using `scan.order`.
+
+The default setting is disabled.
 
 
 [[scan-order]]
 ===== scan.order
 
-Specifies ascending or descending order if `scan.sort` is set to a value other than none. Possible values are asc or desc.
+[]experimental
+
+Specifies ascending or descending order if `scan.sort` is set to a value other than none. Possible values are `asc` or `desc`.
```

Comment:
Change to say: Specifies whether to use ascending or descending order when `scan.sort` is set to a value other than none.